spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkWrite.java

```diff
@@ -229,40 +215,23 @@ private void commitOperation(SnapshotUpdate<?> operation, String description) {
 
   private void abort(WriterCommitMessage[] messages) {
     if (cleanupOnAbort) {
-      Map<String, String> props = table.properties();
-      Tasks.foreach(files(messages))
-          .executeWith(ThreadPools.getWorkerPool())
-          .retry(PropertyUtil.propertyAsInt(props, COMMIT_NUM_RETRIES, COMMIT_NUM_RETRIES_DEFAULT))
```

Comment:
I don't think it is reasonable to use commit retry mechanism for deletes. It is the only place we did this. For now, I added some default configs in `SparkCleanupUtil`. I doubt we want to make it configurable.